simpleble/src/backends/windows/AdapterBase.cpp

```diff
@@ -38,9 +38,20 @@ AdapterBase::AdapterBase(std::string device_id)
         [this](const auto& w, const Advertisement::BluetoothLEAdvertisementReceivedEventArgs args) {
             advertising_data_t data;
             data.mac_address = _mac_address_to_str(args.BluetoothAddress());
+            std::string addr_type;
+            int16_t addr_type_enum = args.BluetoothAddressType();
```

Comment:
Could you use a `switch` statement and test against the implementation-defined enum values? https://learn.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.bluetoothaddresstype?view=winrt-22621